src/domain/image/files/store.ts

```diff
@@ -0,0 +1,7 @@
+
+import fileStorage from '../../../integrations/filestorage/module';
+
+export default async function store(path: string, buffer: Buffer): Promise<void>
+{
+    await fileStorage.writeFile(path, buffer);
```

Comment:
We should check if the file exists, before overwriting it.

Reply:
This is fixed in the `save` function.